source/intro.Rmd

```diff
@@ -297,32 +346,31 @@ tonight at the local cinema; you can look it up on the web or call the cinema
 and ask. But consider by contrast how difficult it is to determine with
 accuracy:
 
-a. what will be the result of more than a hundred million Americans voting for
+1. Whether we will save lives by recommending vitamin D supplements for the
+   whole population as protection against viral infections.  Some evidence
+   suggests that low vitamin D levels predispose to more severe lung
+   infections, and that taking supplements can help [@martineau2017vitamin].
+   But, how certain can we be of the evidence?  How safe are the supplements?
+   Does the benefit, and the risk, differ by ethnicity?
+2. What will be the result of more than a hundred million Americans voting for
    president a month hence; the best attempt usually is a sample of 2000
    people, selected in some fashion or another that is far from random, weeks
    before the election, asked questions that are by no means the same as the
    actual voting act, and so on;
+3. How men feel about women and vice versa.
```

Comment:
This is a slightly odd one; is he hinting at more philosophical type questions?  Can we come up with a better one?

Reply:
No - I think he's thinking of his own field of social science research - surveys, and so on.   And yes - we should.  I'll make that an issue.